Making breadcrumbs clickable (#3515)
fishtown-analytics/docs.getdbt.com: website/docs/docs/cloud/secure/secure-your-tenant.md

---
title: "Secure your tenant"
description: "Learn how to secure your tenant for dbt Cloud"
pagination_next: "docs/cloud/secure/ip-restrictions"
pagination_prev: null
---

<div className="grid--3-col">

<Card
    title="IP restrictions"

>     body="Learn about IP Restrictions and how it can help control which IP addresses are allowed to connect to dbt Cloud."

Author: ```suggestion
    body="Learn about IP Restrictions and how it can help secure your dbt Cloud environment."
```

Reviewer: thank you! good update!
